README.md

```
@@ -1,81 +1,61 @@
 
 #### Prerequisites:
 - Python 3.11+
-- Node.js (for pnpm)
-- pnpm
+- Node.js & pnpm
 
-#### Environment Variables:
-Create a `.env` file in the `backend` directory (or set them directly in your shell):
+#### Backend Setup:
+1. Create a `.env` file in `backend/`:
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)